### PR TITLE
ColumnSearch after click on clear button in IE

### PR DESCRIPTION
**Pull request by yenyen**
Internet Explorer adds a clear button in input. Unfortunately the columnSearch is not trigger when the clear button is pressed.

To fixe this, we have to listen to the 'mouseup' event trigger par the clear button (http://stackoverflow.com/questions/14498396/event-fired-when-clearing-text-input-on-ie10-with-clear-icon)

### CODE REVIEW

File: src/extensions/filter-control/bootstrap-table-filter-control.js [hunk 1]
```diff
@@ -246,6 +246,24 @@
                 }, that.options.searchTimeOut);
             });
 
+            header.off('mouseup', 'input').on('mouseup', 'input', function (event) {
+                var $input = $(this),
+                oldValue = $input.val();
+
+                if (oldValue == "") return;
```
Comment (djhvscf) [inline src/extensions/filter-control/bootstrap-table-filter-control.js]: Please use the equal value and equal type validation and the correct format

if (something === thing) {
...
}

File: src/extensions/filter-control/bootstrap-table-filter-control.js [hunk 2]
```diff
@@ -246,6 +246,24 @@
...
+
+                setTimeout(function(){
+                    var newValue = $input.val();
+
+                    if (newValue == ""){
```
Comment (djhvscf) [inline src/extensions/filter-control/bootstrap-table-filter-control.js]: Please use the equal value and equal type validation

**Comment by yenyen** [comment]
Done ;)

**Comment by djhvscf** [comment]
Great!

**Pull request merged**